zsl/unittest/http.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import json
+from typing import Any
+from typing import Dict
+from typing import Union
+
+from flask.testing import FlaskClient
+from typing import AnyStr
+
+from flask.wrappers import Response
+
+from zsl import inject, Zsl
+from zsl.constants import HttpHeaders, MimeType
+
+
+def json_loads(str_):
+    # type: (AnyStr) -> Dict[str, str]
+    """Parse json from flask response which could be in bytes in Py3."""
+    if isinstance(str_, bytes):
```

Comment:
Use all future imports like in http://python-future.org/quickstart.html#if-you-are-writing-code-from-scratch , are you sure bytes in python 2 is the same type like in python 3? `from builtins import *` ensures that.